zerver/tests/test_realm.py

```diff
@@ -864,6 +864,36 @@ def test_message_retention_days(self) -> None:
         result = self.client_patch("/json/realm", req)
         self.assert_json_success(result)
 
+    def test_jitsi_server_url(self) -> None:
+        self.login("iago")
+        realm = get_realm("zulip")
+        self.assertEqual(realm.video_chat_provider, Realm.VIDEO_CHAT_PROVIDERS["jitsi_meet"]["id"])
+
+        req = dict(jitsi_server_url=orjson.dumps("").decode())
+        result = self.client_patch("/json/realm", req)
+        self.assert_json_error(result, "jitsi_server_url is not an allowed_type")
+
+        req = dict(jitsi_server_url=orjson.dumps("invalidURL").decode())
+        result = self.client_patch("/json/realm", req)
+        self.assert_json_error(result, "jitsi_server_url is not an allowed_type")
+
+        req = dict(jitsi_server_url=orjson.dumps(12).decode())
+        result = self.client_patch("/json/realm", req)
+        self.assert_json_error(result, "jitsi_server_url is not an allowed_type")
```

Comment:
This error message is quite awkward, but I think the right plan for deleting it is for @PIG208 to just migrate this file to the new `@typed_endpoint` system after we merge this.